packages/web-components/fast-components/src/slider/slider.stories.ts

```diff
@@ -9,15 +9,21 @@ FASTSlider;
 FASTDesignSystemProvider;
 
 addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
-    if (name.toLowerCase().startsWith("slider")) {
+    if (name.toLowerCase().endsWith("slider")) {
         const slider1: HTMLElement | null = document.getElementById("switcher");
-        (slider1 as FASTSlider).valueTextFormatter = valueTextFormatter;
+        if (slider1) {
```

Comment:
could this be moved to a function since essentially it's three of the same statements?  This will show up as code duplicate on code climate, so avoiding this increases maintainability.

Reply:
Here's a quick refactor recommendation:

```ts
['switcher', 'switcher2', 'slider1'].forEach(x => {
  const slider = document.getElementById(x);
  slider && (slider as FASTSlider).valueTextFormatter = valueTextFormatter;
});
```

Reply:
code refactor suggestion taken! :)